src/main/java/com/microsoft/azure/search/samples/demo/DemoOperations.java

```diff
@@ -0,0 +1,152 @@
+package com.microsoft.azure.search.samples.demo;
```

Comment:
Try to separate code movement from refactoring. They should be separate commits if possible.